python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1746,6 +1766,32 @@ def findArtifacts(self, templateCoadd, tempExpRefList, imageScalerList):
                              'EDGE': edge})
         return altMasks
 
+    def _prefilterArtifacts(self, spanSetList, exp):
+        """!
+        \brief Remove artifact candidates covered by bad mask plane
```

Comment:
I suggest using numpydoc, even if it doesn't match what's already in the file --- but maybe that's naughty.
My reasoning is that it looks better, doxygen isn't going to parse python for much longer (if it still is), and someone will thank you when it comes time to update.

Reply:
I promise that future person will be me: DM-13587